### PR TITLE
DaskSparseFrame getitem, todense and bugfix

### PR DESCRIPTION
**Pull request by michcio1234**

### CODE REVIEW

File: sparsity/dask/core.py [hunk 1]
```diff
@@ -135,6 +135,22 @@ def index(self):
     def map_partitions(self, func, meta, *args, **kwargs):
         return map_partitions(func, self, meta, *args, **kwargs)
 
+    # noinspection PyTypeChecker
+    def todense(self):
```
Comment (kayibal) [inline sparsity/dask/core.py]: I'd rather use to delayed here, then apply to_dense on each delayed object and use `dd.from_delayed` to construct the dense dask collection.

File: sparsity/dask/core.py [hunk 2]
```diff
@@ -135,6 +135,22 @@ def index(self):
...
+        """Convert into Dask DataFrame or Series
+        
+        Returns
+        -------
+        res: dd.DataFrame | dd.Series
+        """
+        meta = dd.from_pandas(self._meta.todense(), npartitions=1)
```
Comment (kayibal) [inline sparsity/dask/core.py]: Meta could be just `self._meta.todense()`

File: sparsity/sparse_frame.py [hunk 3]
```diff
@@ -1012,8 +1012,10 @@ def reindex_axis(self, labels, axis=0, method=None,
             new_index, idx = self.columns.reindex(labels)
             if idx is None:
                 return self.copy()
-            # we have a hidden zero column to replace missing indices (-1)
-            new_data = self._data.T[idx].T[:-1]
```
Comment (kayibal) [inline sparsity/sparse_frame.py]: This should work on empty frames if scipy>=1.0.0

File: sparsity/dask/core.py (hunk 2 above)
Comment (michcio1234) [reply sparsity/dask/core.py]: Not really, we need dask object here

**Pull request merged**